readthedocs/projects/forms.py

```diff
@@ -667,6 +667,7 @@ class Meta:
         )
         labels = {
             "enabled": _("Enable Addons"),
+            "doc_diff_enabled": _("Visual diff enabled"),
```

Comment:
Looks good. We should probably update the docs too, but that can be another PR 🙃